src/main/java/com/cgdpd/library/dto/pagination/SortParam.java

```diff
@@ -0,0 +1,16 @@
+package com.cgdpd.library.dto.pagination;
+
+import org.springframework.data.domain.Sort;
+
+public record SortParam(String field, Order order) {
```

Comment:
Add one or two tests to check if pagination with sort is working
And add validations in the constructor to check if values are not invalid